plugins/other/sshd_log

```diff
@@ -28,18 +29,35 @@
 #env.logfile /var/log/messages
 #env.category users
 #
+# config example with journald
+#[sshd_log]
+#group systemd-journal
+#env.logfile journald
+#env.journald_unit sshd.service
```

Comment:
what do you think about merging these two settings (logfile + journald_unit) into a single one - e.g. `env.logfile journald:sshd.service`?